src/EnergyPlus/PlantLoopHeatPumpEIR.cc

```diff
@@ -287,6 +287,7 @@ void EIRPlantLoopHeatPump::doPhysics(EnergyPlusData &state, Real64 currentLoad)
 
     // evaluate the actual current operating load side heat transfer rate
     auto &thisLoadPlantLoop = state.dataPlnt->PlantLoop(this->loadSidePlantLoc.loopNum);
+    auto &thisSourcePlantLoop = state.dataPlnt->PlantLoop(this->sourceSidePlantLoc.loopNum);
```

Comment:
Not sure this is safe. If not water-cooled, sourceSidePlantLoc.loopNum will be 0. In the new plant work I put this down at line 355.